settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'spring-authorisation-server'
```

Comment:
Even though "authorisation" and "authorization" are both legitimate spellings, will you please change this to "spring-authorization-server" (with a 'z') so that it lines up with the project name?